gaeguli/target.h

```diff
@@ -75,6 +75,8 @@ const gchar            *gaeguli_target_get_peer_address
 
 GVariant               *gaeguli_target_get_stats      (GaeguliTarget       *self);
 
+GstElement             *gaeguli_target_get_sink       (GaeguliTarget       *self);
+
```

Comment:
I am unsure if we can ask another GStreamer dependency to an application by exposing GstElement.

Reply:
As it's implemented now, `GaeguliTarget` creates its own stream adaptor internally, where srtsink etc. are indeed accessible. See [here](https://github.com/hwangsaeul/gaeguli/pull/120/#issuecomment-738036143) for my suggestion how to connect to the adaptor's signal form the application. If done that way, `gaeguli_target_get_sink ()` will not be necessary.